### DISCUSSION TITLE
Issue in fiiling OTP Field

### DISCUSSION THREAD

**Discussion by abhishekrawe** [pkovzz/ngx-otp-input#15] [Q&A] [answered]
I face an issue while copy pasting the OTP via clipboard on mobile -

I copy the OTP and paste is by clicking the clipboard it filled only one space, for eg I have 6 slots for OTP is filed only 1. 
it works fine for copy paste at space and by typing but selecting by clipboard it gives me an error.

rather than filling or pasting the OTP, do you have any solution for automating this like it filled automatic?

**Comment by pkovzz** [comment, marked as answer]
Hi!

I'm not sure I fully understand what you are asking for. Do you wish some kind of mechanism for setting value, like a dedicated `setter` function?

**Comment by Anjan-12** [comment]
your packages is working fine in web version but when i try to use in ionic app it doesnot work .why?
